tests/interfaces/test_classify_image_supervised.py

```diff
@@ -0,0 +1,98 @@
+from typing import Any, Dict, Hashable, Iterable, Iterator, List, Mapping, Sequence
+import unittest
+import unittest.mock as mock
+import numpy as np
+
+from smqtk_classifier import ClassifyImageSupervised
+from smqtk_classifier.interfaces.classification_element import CLASSIFICATION_DICT_T
+from smqtk_classifier.interfaces.classify_image import IMAGE_ITER_T
+from smqtk_classifier.exceptions import ExistingModelError
+
+
+class DummySupervisedClassifier (ClassifyImageSupervised):
+
+    EXPECTED_LABELS = ['constant']
+    EXPECTED_HAS_MODEL = False
+
+    @classmethod
+    def is_usable(cls) -> bool:
```

Comment:
This should not need an override as the default implementation at the `Pluggable` interface level should return True.